packages/mobile/src/exchange/CeloGoldHistoryChart.tsx

```diff
@@ -300,30 +265,9 @@ const styles = StyleSheet.create({
   container: {
     marginBottom: 0,
   },
-  goldPrice: {
-    padding: variables.contentPadding,
-  },
-  goldPriceTitle: {
-    ...fontStyles.h2,
-    marginBottom: 8,
-  },
-  goldPriceValues: { flexDirection: 'row', alignItems: 'flex-end' },
-  goldPriceCurrentValue: {
-    ...fontStyles.mediumNumber,
-  },
-  goldPriceWentUp: {
-    ...fontStyles.regular,
-    color: colors.greenUI,
-  },
-  goldPriceWentDown: {
-    ...fontStyles.regular,
-    marginBottom: 2, // vertically align with the current price
-    marginLeft: 4,
-    color: colors.warning,
-  },
   loader: {
     width: CHART_WIDTH,
-    height: CHART_HEIGHT + 130,
+    height: CHART_HEIGHT + 37.5,
```

Comment:
Well, this is the size of the block with Gold exchange rate. It is needed, so after loader disappears - buttons below it would not flicker. I've found it using binary search, and neither 37 nor 38 were not good enough 😄